action.yml

```diff
@@ -127,7 +127,7 @@ runs:
 
     - name: Install nix
       if: inputs.skip-nix-installation == 'false'
-      uses: DeterminateSystems/nix-installer-action@v4
+      uses: DeterminateSystems/nix-installer-action@65d7c888b2778e8cf30a07a88422ccb23499bfb8  # v4
```

Comment:
Dependabot is smart enough to automatically update `# vX` comments, so this won't fall out of date.  See https://github.com/dependabot/dependabot-core/pull/5951

65d7c888b2778e8cf30a07a88422ccb23499bfb8 is the SHA that the v4 tag points at:

https://github.com/DeterminateSystems/nix-installer-action/releases/tag/v4